AirCasting/Dashboard/DashboardView.swift

```diff
@@ -28,12 +29,15 @@ struct DashboardView: View {
         coreDataHook.sessions
     }
 
-    init(coreDataHook: CoreDataHook) {
+    init(coreDataHook: CoreDataHook, measurementsDownloadingInProgress: Binding<Bool>) {
         let navBarAppearance = UINavigationBar.appearance()
         navBarAppearance.largeTitleTextAttributes = [.foregroundColor: UIColor(Color.darkBlue)]
         _coreDataHook = StateObject(wrappedValue: coreDataHook)
         self.sessionSynchronizer = Resolver.resolve(SessionSynchronizer.self)
         _isRefreshing = .init(wrappedValue: sessionSynchronizer.syncInProgress.value)
+        _measurementsDownloadingInProgress = .init(projectedValue: measurementsDownloadingInProgress)
+        var log = isRefreshing
+        Log.info("##### IS REFRESHING: \(log)")
```

Comment:
Should we keep that log? If so can we remove those # characters?

Reply:
Sorry, this was left by mistake